client/src/Utils/rfFunctions.ts

```diff
@@ -0,0 +1,20 @@
+export function calculateDate(start: Date, count: number, sampleRate: number) {
+  let time = start.getTime() + count / sampleRate;
+  return new Date(time).toISOString();
+}
+
+export function calculateFrequency(freq: number) {
+  if (freq.toString().length >= 10) {
```

Comment:
just a style thing it probably doesnt matter but I would probably do > 999_999_999